bndtools.m2e.runtime/bnd.bnd

```diff
@@ -0,0 +1,7 @@
+-resourceonly: true
+
+Bundle-SymbolicName: bndtools.m2e.runtime; singleton:=true
```

Comment:
Will rename it to bndtools.debug.fragment, so it is clear that it is 1) only used for debug 2) only a fragment bundle